### PR TITLE
Added endpoint to remove users.

### PR DESCRIPTION
**Pull request by andremrezende**
Fixed other failed integration tests

### CODE REVIEW

**Comment by ssouza** [comment]
@andremrezende tem conflitos a resolver

**Review by ssouza — COMMENTED** [review]
Resolver conflitos

File: src/test/java/br/com/senior/core/BaseIT.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 public class BaseIT {
 
     protected LoginOutput login() throws ServiceException {
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/BaseIT.java]:
```suggestion
    protected static LoginOutput login() throws ServiceException {
```

File: src/test/java/br/com/senior/core/BaseIT.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
-        return login(System.getenv("username"), System.getenv("password_valid"));
+        return login(System.getenv("SENIOR_USERNAME"), System.getenv("PASS"));
     }
 
     protected LoginOutput login(String username, String password) throws ServiceException {
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/BaseIT.java]:
```suggestion
    protected static LoginOutput login(String username, String password) throws ServiceException {
```

File: src/test/java/br/com/senior/core/authentication/AuthenticationIT.java [hunk 3]
```diff
@@ -21,7 +21,7 @@ public void testValidLogin() throws ServiceException {
 
     @Test
     public void testInvalidLogin() {
-        LoginInput input = new LoginInput(System.getenv("USERNAME"), System.getenv("PASSWORD_INVALID"));
+        LoginInput input = new LoginInput(System.getenv("SENIOR_USERNAME"), System.getenv("PASSWORD_INVALID"));
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/authentication/AuthenticationIT.java]:
```suggestion
        LoginInput input = login(System.getenv("SENIOR_USERNAME"), System.getenv("PASSWORD_INVALID"));
```

File: src/test/java/br/com/senior/core/authorization/AuthorizationIT.java [hunk 4]
```diff
@@ -21,7 +21,7 @@
 
     @BeforeClass
     public static void beforeClass() throws ServiceException {
-        token = new AuthenticationClient().login(new LoginInput(System.getenv("username"), System.getenv("password_valid"))).getJsonToken().getAccess_token();
+        token = new AuthenticationClient().login(new LoginInput(System.getenv("SENIOR_USERNAME"), System.getenv("password_valid"))).getJsonToken().getAccess_token();
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/authorization/AuthorizationIT.java]:
```suggestion
        token = login().getJsonToken().getAccess_token();
```

File: src/test/java/br/com/senior/core/notification/NotificationIT.java [hunk 5]
```diff
@@ -23,7 +23,7 @@
 
     @BeforeClass
     public static void beforeClass() throws ServiceException {
-        token = new AuthenticationClient().login(new LoginInput(System.getenv("username"), System.getenv("password_valid"))).getJsonToken().getAccess_token();
+        token = new AuthenticationClient().login(new LoginInput(System.getenv("SENIOR_USERNAME"), System.getenv("PASS"))).getJsonToken().getAccess_token();
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/notification/NotificationIT.java]:
```suggestion
        token = login().getJsonToken().getAccess_token();
```

File: src/test/java/br/com/senior/core/user/UserIT.java [hunk 6]
```diff
@@ -47,83 +53,167 @@
 
     @BeforeClass
     public static void beforeClass() throws ServiceException {
-        usernameExpected = Arrays.stream(System.getenv("USERNAME").split("@")).findFirst().orElse(null);
-        token = new AuthenticationClient().login(new LoginInput(System.getenv("username"), System.getenv("password_valid"))).getJsonToken().getAccess_token();
+        usernameExpected = Arrays.stream(System.getenv("SENIOR_USERNAME").split("@")).findFirst().orElse(null);
+        token = new AuthenticationClient().login(new LoginInput(System.getenv("SENIOR_USERNAME"), System.getenv("PASS"))).getJsonToken().getAccess_token();
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/user/UserIT.java]:
```suggestion
        token = login().getJsonToken().getAccess_token();
```

**Review by andremrezende — COMMENTED** [review]
Ajustado

File: src/test/java/br/com/senior/core/authentication/AuthenticationIT.java [hunk 7]
```diff
@@ -21,7 +21,7 @@ public void testValidLogin() throws ServiceException {
...
         Assert.assertThrows(ServiceException.class, () -> new AuthenticationClient().login(input));
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/authentication/AuthenticationIT.java]:
```suggestion
        Assert.assertThrows(ServiceException.class, () -> login(System.getenv("SENIOR_USERNAME"), System.getenv("PASSWORD_INVALID")));
```

File: src/test/java/br/com/senior/core/authentication/AuthenticationIT.java (hunk 3 above)
Comment (ssouza) [inline src/test/java/br/com/senior/core/authentication/AuthenticationIT.java]:
```suggestion
```

File: src/test/java/br/com/senior/core/authentication/AuthenticationIT.java [hunk 8]
```diff
@@ -21,8 +21,8 @@ public void testValidLogin() throws ServiceException {
 
     @Test
     public void testInvalidLogin() {
-        LoginInput input = new LoginInput(System.getenv("USERNAME"), System.getenv("PASSWORD_INVALID"));
-        Assert.assertThrows(ServiceException.class, () -> new AuthenticationClient().login(input));
+        LoginInput input = login(System.getenv("SENIOR_USERNAME"), System.getenv("PASSWORD_INVALID"));
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/authentication/AuthenticationIT.java]:
```suggestion
```

File: src/test/java/br/com/senior/core/authorization/AuthorizationIT.java [hunk 9]
```diff
@@ -21,7 +21,7 @@
```
Comment (ssouza) [inline src/test/java/br/com/senior/core/authorization/AuthorizationIT.java:42]: Remover os imports desnecessários:
import br.com.senior.core.authentication.AuthenticationClient;
import br.com.senior.core.authentication.pojos.LoginInput;

File: README.md [hunk 10]
```diff
@@ -40,6 +40,13 @@ Utilizando como exemplo o login, caso a aplicação queira efetuar o login integ
     LoginOutput output = client.login(new LoginInput(username, password));
     ...
 ```
+Definir as seguintes variáveis de ambiente:
+
+```
+SENIOR_USERNAME=<usuario>
+PASS=<senha_do_usuario>
+TENANT_NAME=<nome_do_tenant>
+```
```
Comment (ssouza) [inline README.md:54]: Não deveria ter as outras variáveis?

- USERNAME_MFA
- PASSWORD_MFA
- ACCESS_KEY
- SECRET
- TENANT_DOMAIN

**Review by ssouza — APPROVED** [review]

**Pull request merged**